src/lib/Form/Type/Content/ContentFieldType.php

```diff
@@ -46,12 +46,13 @@ public function configureOptions(OptionsResolver $resolver)
             ->setDefaults([
                 'content' => null,
                 'location' => null,
-                'contentCreateStruct' => null,
-                'contentUpdateStruct' => null,
+                'contentCreateStruct' => null, // deprecated
+                'contentUpdateStruct' => null, // deprecated
```

Comment:
Use https://symfony.com/doc/5.x/components/options_resolver.html#deprecating-the-option instead.